### DISCUSSION TITLE
Can't change nvim-treesitter configuration.

### DISCUSSION THREAD

**Discussion by TheUtopian** [LazyVim/LazyVim#703] [Q&A] [answered]
I created a file inside "plugins" folder and added:
`
return {
  {
    "nvim-treesitter/nvim-treesitter",
    opts = {
      highlight = {
        enable = false,
      },
      is_supported = function()
        return false
      end,
    },
  }
}
`

But it shows me error: **"Two plugins with the same name and different url:"**,
I've tried to change name to "nvim-treesitter/nvim-treesitter.configs" but then it says that there's no **"configs"** module.
What name should I use to make it work properly?

**Comment by folke** [comment, marked as answer]
check your code. It's pretty obvious. typo

**Reply by TheUtopian** [reply]
Thanks, the error is gone, but it still dosen't turn off the highlighting. As I understood there should be nvim-treesitter.configs module but Idk how to choose it for lazyvim.

**Reply by folke** [reply]
Looking at the nvim-treesitter docs, it is nowhere mentioned you can use the is_supported that way.
https://github.com/nvim-treesitter/nvim-treesitter#adding-modules
